docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md

```diff
@@ -101,3 +101,24 @@ For any third-party Cron scheduling tools, you can disable Crontab and add the f
 
 * Set `vm.swappiness` to 0.
 * Set `mount` path permissions to 0755.
+
+{{< note title="Note" >}}
+By default, YugabyteDB Anywhere uses SSH via OpenSSH. YugabyteDB Anywhere also supports the use of SSH2 via Tectia SSH. For more information, see [Enable Tectia SSH](#enable-tectia-ssh).
```

Comment:
OpenSSH is also based on SSH2. I think we can mention something like, `By default, YugabyteDB Anywhere uses OpenSSH for ssh to remote nodes. YugabyteDB Anywhere also supports the use of Tectia SSH that is based on the latest SSH G3 protocol. For more information, see`